### ISSUE TITLE
when support input with tab?

### ISSUE THREAD

**Issue by gs80140** [spring-projects/spring-shell#59]
such as this command import --T file --name "product" --table "product" --d \t

When I enter the \t look into the tab key ,i have an error "You should specify a default option d"

**Comment by ericbottard** [comment]
Not sure I understand your request, but I think you should try this:
```
import --T file --name "product" --table "product" --d '\t'
```

**Comment by gs80140** [comment]
thanks for your response,but i am sorry to tell you that I still got this message "You should specify value for option 'd' for this command"

'\t' can't be analyzed by spring shell 1.1.0 org.springframework.shell.core.SimpleParser on line 220

boolean specifiedKey = !StringUtils.hasText(value) && options.containsKey(sourcedFrom);

'\t' is not text , isWhitespace 

so i want to say ,spring shell command can't input control character
am i right?can you understand me?

thanks again!!!

**Comment by ericbottard** [comment]
Confirmed. Created https://jira.spring.io/browse/SHL-170 to track this

**Comment by ericbottard** [comment]
FYI, issue has been fixed in 511a7e2
